PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,20 @@
 Closes #ISSUE
 
-_Description:_
+**Description:**
 
 <!-- Briefly describe what you're adding or fixing with this PR -->
 
-❤️ Thank you!
+Thank you! 🚀
 
 ---
 
-<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
-v                    Thanks for creating a PR!
-v    Before smashing the submit button please review the checkboxes
-☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
+For contibutor:
 
 - [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
 - [ ] Reviewed `Files changed` in the github PR explorer
+- [ ] Added snapshots of the view components
```

Comment:
doesn't our CI check this for us?

Reply:
snapshots on the PR description, not of the HTML. Will update it

Reply:
screenshot not snapshot I suppose?